source/extensions/filters/http/gcp_authn/gcp_authn_filter.cc

```diff
@@ -12,8 +14,14 @@ namespace HttpFilters {
 namespace GcpAuthn {
 
 using ::Envoy::Router::RouteConstSharedPtr;
+using ::google::jwt_verify::Status;
 using Http::FilterHeadersStatus;
 
+void addTokenToRequest(Http::RequestHeaderMap& hdrs, absl::string_view token_str) {
```

Comment:
In general consider putting free functions that are not used in other .cc files in anonymous namespace. This will avoid the work the linker needs to do to export it.